pages/timeline.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Timeline
-layout: timeline
+layout: timeline_edtf
```

Comment:
Does the old layout still work with your regex magic?

Reply:
The old layout still works.